containers/codespaces-linux/.devcontainer/base.Dockerfile

```diff
@@ -68,8 +68,13 @@ RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
     && mkdir -p /usr/local/etc/vscode-dev-containers/ \
     && mv -f /tmp/scripts/first-run-notice.txt /usr/local/etc/vscode-dev-containers/
 
+# Remove existing Python installation from Oryx base image
+RUN rm -rf /opt/python && rm "${PYTHON_ROOT}/current"
+
 # Install Python, JupyterLab, common machine learning packages, and Ruby utilities
-RUN bash /tmp/scripts/python-debian.sh "none" "/opt/python/latest" "${PIPX_HOME}" "${USERNAME}" "true" \
+RUN bash /tmp/scripts/python-debian.sh "3.10.4" "/opt/python/3.10.4" "${PIPX_HOME}" "${USERNAME}" "true" "true" "true" \
```

Comment:
```suggestion
RUN bash /tmp/scripts/python-debian.sh "3.10.4" "/opt/python/3.10.4" "${PIPX_HOME}" "${USERNAME}" "true" "true" "false" \
```

Installing with Oryx reintroduces the `lzma` bug 😞:

<img width="514" alt="ModuleNotFoundError: No module named '_lzma'" src="https://user-images.githubusercontent.com/19893438/181375831-bc871ccb-7f7b-4e53-ab37-5198689d8062.png">

We need to build Python from source to fix `lzma`